themes/default/content/resources/how-to-build-azure-serverless-application/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "How do build a serverless applications with Azure, blob storage and API management"
+allow_long_title: true
+meta_desc: "Learn to build a serverless applications with this step-by-step guide! Learn how to use Azure Functions for compute, Blob Storage for data, and API Management to expose your APIs. Ideal for developers and architects aiming for efficient, scalable solutions."
+type: ai-answers
+date: 2023-07-24
+---
+
+
+## How do build a serverless applications with Azure, blob storage and API management
```

Comment:
Ditto ... also, is the casing coming from us?  (iow Blob Storage, like above, rather than blob storage)

Reply:
To add more detail, I took straight prompts from Pulumi AI and created these example, there’s no testing and automation and definitely errors in here. The goal of the experiment is to see if we can generate traffic, rank on search and provide useful information to people, with a caveat that its AI created and will take more testing and validation.

We’ll need to continue to improve the quality of results and that can be explored later on. for now, we have an easy way for people to leave feedback for us